main/wallet-backend.js

```diff
@@ -286,6 +292,29 @@ class WalletBackend {
     this.transactions = transactions
     this.onTransactionUpdate()
   }
+
+  /**
+   * @returns {Promise<ethers.BigNumber>}
+   */
+  async fetchScheduledRewards () {
+    assert(this.address, 'address')
+    assert(this.meridian, 'meridian client')
+    try {
+      return await this.meridian.rewardsScheduledFor(this.address)
+    } catch (/** @type {any} */ err) {
+      if (
+        err?.error?.error instanceof Error &&
+        err.error.error.message.includes(
+          `resolve address ${this.addressDelegated}: actor not found`
+        )
+      ) {
+        // no-op, our address was not assigned any rewards yet
```

Comment:
See the discussion in Slack: 
https://filecoinproject.slack.com/archives/C03PVLW721L/p1699433434940039

When calling `rewardsScheduledFor(address)`, I am getting the following error, where the delegated address `f410[redacted]` is my Station Wallet.

```
CallWithGas failed: 
 call raw get actor: 
  resolution lookup failed (f410[redacted]): 
   resolve address f410[redacted]: 
    actor not found
```

Reply:
Fixed in [ea21ce3](https://github.com/filecoin-station/desktop/pull/1082/commits/ea21ce3c980fc64bb3dde90903d0a6f0c06a19df)